tests/unit/type/stream-cgroups.tcl

```diff
@@ -1048,6 +1089,21 @@ start_server {
         assert_equal [dict get $group lag] 2
     }
 
+    test {Loading from legacy (Redis <= v7.0.x, rdb_ver < 11) persistence} {
```

Comment:
https://github.com/redis/redis/actions/runs/3588151642/jobs/6039227428#step:7:700
the test only failed in centos (my local centos also failed), with `ERR DUMP payload version or checksum are wrong.`

after setting `DEBUG SET-SKIP-CHECKSUM-VALIDATION 1`
the test above it, the one `Loading from legacy (Redis <= v6.2.x, rdb_ver < 10) persistence` failed, with `ERR Bad data format`